poc_mobile_verifier/src/verifier.rs

```diff
@@ -171,9 +187,9 @@ impl Verifier {
     }
 }
 
-pub struct VerifiedEpoch {
-    pub heartbeats: Vec<Heartbeat>,
-    pub speedtests: SpeedtestAverages,
+pub struct VerifiedEpoch<H, S> {
```

Comment:
huh.. couldn't declare them as the actual Stream Impls they're going to be?

Reply:
They can't be named since their type decls include closures. Unless we used a boxed stream, but I wanted to see how far I could get without boxing